README.md

```diff
@@ -17,6 +17,18 @@ But if you want to **contribute** and make this much better for other developer
 
 If you created something awesome and want to contribute then feel free to open an [pull request](https://github.com/saadpasta/developerFolio/pulls).
 
+## Table of Contents
+- [Sections](#sections)\
+- [Getting Started](#getting-started)\
+- [How to Use](#how-to-use)\
+- [Github Setup](#github-setup-for-open-source-projects)\
```

Comment:
We should change this to `Linking Portfolio to GitHub` and appropriately change the heading of the section to something similar too.